tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Controllers/OrderItemsControllerTests.cs

```diff
@@ -23,13 +25,13 @@ namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Controllers
     internal sealed class OrderItemsControllerTests
     {
         [Test]
-        public void Constructor_OrderRepositoryNull_NullException()
+        public void Constructor_NullOrderRepository_NullException()
         {
             var context = OrderItemsControllerTestContext.Setup();
             
             Assert.Throws<ArgumentNullException>(() =>
             {
-                var _ = new OrderItemsController(null, context.UpdateOrderItemService.Object);
+                var _ = new OrderItemsController(null, context.UpdateOrderItemServiceMock.Object, Mock.Of<ICreateOrderItemService>());
```

Comment:
Could use the `OrderItemsControllerBuilder` here so with the introduction of new dependencies the test code would only have one place to change. Very minor so use best judgement on this.

Reply:
Will do, I like that, it will help with merges also